components/camel-irc/src/main/java/org/apache/camel/component/irc/IrcConfiguration.java

```diff
@@ -44,7 +44,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(IrcConfiguration.class);
 
     private boolean usingSSL;
-    private List<IrcChannel> channels = new ArrayList<>();
+    private List<IrcChannel> channelList = new ArrayList<>();
```

Comment:
IMHO is getting confusing now among "channels", "channelList", "getListOfChannels", "getChannelsList". For me they all sound the same, wouldn't make sense to rethink of the namings here? 
IMHO it could be something like this: 
`getChannels` which returns `channels`
`getListOfChannels` this theoretically could return `channelList` but now returns channels string without pwd which is a little bit confusing I guess.

Reply:
@omarsmak I'll try to simplify these names, I agree that it could be misleading